### PR TITLE
[DNS] PROBE_POSSIBLE troubleshooting page and update to plans.json

### PR DESCRIPTION
**Pull request by RebeccaTamachiro**
PCX-7084
PCX-7410

### CODE REVIEW

File: content/dns/_partials/_dns-errors-ts-action.md [hunk 1]
```diff
@@ -0,0 +1,15 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+inputParameters: param1
```
Comment (pedrosousa) [inline content/dns/_partials/_dns-errors-ts-action.md]:
```suggestion
inputParameters: errorName
```
Not really necessary, but it helps to have a descriptive parameter name.

File: content/dns/_partials/_dns-errors-ts-intro.md [hunk 2]
```diff
@@ -0,0 +1,15 @@
+---
+_build:
+  publishResources: false
+  render: never
+  list: never
+inputParameters: param1
```
Comment (pedrosousa) [inline content/dns/_partials/_dns-errors-ts-intro.md]:
```suggestion
inputParameters: errorName
```

File: data/plans.json [hunk 3]
```diff
@@ -739,6 +739,20 @@
           "ent": "Yes"
         }
       }
+    },
+    "reverse_zones": {
+      "title": "Reverse Zones",
```
Comment (pedrosousa) [inline data/plans.json]:
```suggestion
      "title": "Reverse zones",
```

**Review by pedrosousa — APPROVED** [review]

File: content/dns/_partials/_dns-errors-ts-action.md (hunk 1 above)
Comment (RebeccaTamachiro) [reply content/dns/_partials/_dns-errors-ts-action.md]: Oh, that's a nice touch indeed. I hadn't realized we could do that.
Thanks, Pedro!

**Pull request merged**